cartography/intel/aws/__init__.py

```diff
@@ -108,6 +150,10 @@ def _sync_multiple_accounts(neo4j_session, accounts, sync_tag, common_job_parame
     run_cleanup_job('aws_post_ingestion_dns_cleanup.json', neo4j_session, common_job_parameters)
 
 
+def _parse_aws_requested_syncs(aws_requested_syncs: str) -> List[str]:
+    return aws_requested_syncs.split(',')
```

Comment:
in case someone puts spaces between the items e.g. "iam, s3,       ec2" maybe we could do something like 
```suggestion
    return [x.strip() for x in aws_requested_syncs.split(',')]
```
?